doc/developer/building-frr-for-ubuntu1804.rst

```diff
@@ -104,6 +104,10 @@ And load the kernel modules on the running system:
 
    sudo modprobe mpls-router mpls-iptunnel
 
+If the above command returns an error, you may need to install the appropriate
+or latest linux-modules-extra-<kernel-version>-generic package. For example
+``apt-get install linux-modules-extra-4.15.0-48-generic``
```

Comment:
maybe we could use ``linux-modules-extra-`uname -r` `` here instead.

Reply:
Good point.